src/common/cryptoutils.cpp

```diff
@@ -205,13 +205,74 @@ RetWithError<SharedPtr<crypto::PrivateKeyItf>> CertLoader::LoadPrivKeyFromFile(c
     return mCryptoProvider->PEMToX509PrivKey(*buff);
 }
 
+/***********************************************************************************************************************
+ * Private
+ **********************************************************************************************************************/
+
+RetWithError<size_t> FindSubstr(const String& src, size_t startPos, const String& substr)
```

Comment:
Could we have FindSubstr and FindAny as public string mehtods? It would be useful for other as well. And  add small unit test.

Reply:
moved to String class.